packages/node_modules/@ciscospark/widget-recents/src/components/NoSpaces.css

```diff
@@ -0,0 +1,30 @@
+.noSpacesWrapper {
+  height: 100%;
+  text-align: center;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  justify-content: center;
+  background-color: black;
+}
+
+.noSpacesImg {
+  height: 72px;
+  width: 100%;
+  margin-bottom: 24px;
+}
+
+.noSpacesTitle {
+  color: #fff;
+  font-family: CiscoSansTT Bold,Helvetica Neue,Arial;
```

Comment:
Could we use a collab-ui `h2` to give us most of this styling (aside from the alignment and padding)?

Reply:
will try it and get back to you